auth_quick_master/static/description/index.html

```diff
@@ -4,6 +4,8 @@
             <h2 class="oe_slogan" style="color:#875A7B;">Quick Auth (Master)</h2>
             <h3 class="oe_slogan">Authentication provider for <a href="https://apps.odoo.com/apps/modules/12.0/auth_quick/">Quick Authentication</a> module</h3>
         </div>
+        <br/>
+        <font style="font-size: 120%;">Follow <i>Documentation</i> tab to see Configuration details</font>
```

Comment:
@Mirgalimova 
Follow the <b>Documentation</b> tab for the configuration details